internal/handlers/handler_verify.go

```diff
@@ -180,13 +180,15 @@ func verifySessionCookie(ctx *middlewares.AutheliaCtx, targetURL *url.URL, userS
 		if err == authentication.ErrUserNotFound {
 			err = ctx.Providers.SessionProvider.DestroySession(ctx.RequestCtx)
 			if err != nil {
-				ctx.Logger.Error(fmt.Errorf("Unable to destroy user session after provider refresh didn't find the user: %s", err))
+				ctx.Logger.Error(fmt.Errorf("unable to destroy user session after provider refresh didn't find the user: %s", err))
```

Comment:
```suggestion
				ctx.Logger.Error(fmt.Errorf("Unable to destroy user session after provider refresh didn't find the user: %s", err))
```

Reply:
this one was expected to be lower case because we create an error.